pyproject.toml

```diff
@@ -18,14 +18,18 @@ repository = "https://github.com/moj-analytical-services/mojap-arrow-pd-parser"
 homepage = "https://github.com/moj-analytical-services/mojap-arrow-pd-parser"
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<=3.9"
+python = ">=3.7.1,<3.10.0"
 pandas = "^1.2"
-pyarrow = ">=2.0.0,<4.0.0"
+pyarrow = ">=2.0.0,<6.0.0"
```

Comment:
Might need to lockdown the lower bound of the pyarrow dependency more (will test this)